### PR TITLE
[plugin.video.tv3.cat] 1.4.1

### PR DESCRIPTION
**Pull request by jqandreu**
### Description

Fixed web server changes. Fixed errors listing videos in two sections
### Checklist:

- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.tv3.cat/changelog.txt [hunk 1]
```diff
@@ -1,4 +1,9 @@
-﻿v1.4.0
+﻿v1.4.1
+- Corregits canvis al website. Corregits errors en les seccions "Avui destaquem" i "Col·leccions"
```
Comment (enen92) [inline plugin.video.tv3.cat/changelog.txt]: Changelog in gotham is presented to the user. Please keep it in english only.

Comment (jqandreu) [reply plugin.video.tv3.cat/changelog.txt]: Ok, I will fix it today

**Comment by jqandreu** [comment]
Modified changelog to english only. I have a question, the main users of this plugin don't speak english, they speak catalan. Why changelog must be in english only when some plugins are from international channels which users don't read and don't speak english?

**Comment by enen92** [comment]
@jqandreu kodi's default language is en_GB. You can then filter add-ons by language but any info that is presented to the user must be always in english

**Comment by jqandreu** [comment]
Ok, thanks

**Comment by Rechi** [comment]
@jqandreu this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Comment by jqandreu** [comment]
I will make Python 3 compatible in the next weeks

**Pull request merged**